torch/_functorch/aot_autograd.py

```diff
@@ -2982,6 +2986,17 @@ def backward(ctx, *flat_args):
             del contiguous_args
 
             def call_compiled_backward():
+                if ctx._compiled_autograd_tracing:
+                    # For compiled autograd, run raw FX graph so that it can be inlined into the larger graph
+                    symints = ctx._compiled_autograd_symints
+                    assert len(symints) == len(ctx.symints)
+                    all_args[:len(symints)] = symints
+                    context = torch._C._DisableAutocast if disable_amp else nullcontext
+                    with context():
+                        out = normalize_as_list(bw_module(*all_args))
```

Comment:
Duplicating the logic here from below sounds like a painful divergence to maintain going forward? maybe we want to remove that duplication? I'll let @bdhirsh weight in on this one